ivy/functional/backends/jax/elementwise.py

```diff
@@ -12,7 +12,13 @@
 from . import backend_version
 
 
-def abs(x: Union[float, JaxArray], /, *, out: Optional[JaxArray] = None) -> JaxArray:
+def abs(
+    x: Union[float, JaxArray],
+    /,
+    *,
+    where: Union[bool, JaxArray] = None,
```

Comment:
The default value should be `True` instead of `None`, just like the other backend implementations